airflow/cli/commands/task_command.py

```diff
@@ -213,6 +213,29 @@ def task_list(args, dag=None):
         print("\n".join(sorted(tasks)))
 
 
+@cli_utils.action_logging
+def task_states_for_dag_run(args):
+    """Get the status of all task instances in a dag run"""
+    if not args.dag_id or not args.execution_date:
```

Comment:
Is this condition needed? dags_id and execution are required arguments. I think we already have one higher level validation

Reply:
fixed